docs/snapshotting/versioning.md

```diff
@@ -19,15 +33,15 @@ Firecracker persists the microVM state as 2 separate objects:
 *The block devices attached to the microVM are not considered part of the state and need to be managed separately.*
 
 ### Guest memory
-The guest memory file is agnostic to Firecracker's version tolerant save/restore implementation as its contents are fully owned by the guest OS. The microVM memory is by choice saved as a dump of each page or as a sparse file of guest modified pages only. 
+The guest memory file contains the microVM memory saved as a dump of all pages.
+
 ### MicroVM state
-The microVM state file stores the internal state of the VMM (device emulation, KVM and vCPUs) with 2 exceptions - serial emulation and vsock backend. 
+In the VM state file, Firecracker stores the internal state of the VMM (device emulation, KVM and vCPUs) with 2 exceptions - serial emulation and vsock backend. 
 
-While we continuously improve and extend Firecracker's features by adding new capabilities, devices or enhancements, the microVM state file may change both structurally and semantically with each new release. The state file includes versioning information and each Firecracker release implements distinct save/load logic for the supported version space. 
+Since we continuously improve and extend Firecracker's features by adding new options, devices or enhancements, the VM state file may change both structurally and semantically with each new release. The state file includes versioning information and each Firecracker release implements distinct save/restore logic for the supported version space.
```

Comment:
Except save/load -> save/restore I would say that the doc was better before the change

Reply:
done

Reply:
hmm I think @raduiliescu was suggesting to dismiss new changes, except for the `save/load` part, am i right?